Fixed #26615 -- Made password reset token invalidate when changing email. (#13551)
django/django: docs/releases/3.2.txt

  constrained environments. If this is the case, the existing hasher can be
  subclassed to override the defaults.

> * Password reset tokens now incorporate the user's email address, if available.
>   Altering the user's email will now invalidate existing reset tokens.

Reviewer: Maybe:
```suggestion
* The password reset mechanism now invalidates a token when the user email is
  changed.
```

That's not a new feature, we should move this to the `Backwards incompatible changes in 3.2 -> Miscellaneous`.

  ``False`` if the file cannot be locked, instead of raising
  :exc:`BlockingIOError`.

> * The password reset mechanism now invalidates a token when the user email is

Reviewer 2: `invalidates tokens` (in general)?

Reviewer: True, thanks :+1:
